NetTopologySuite.Samples.Console/Operation/Polygonize/SplitPolygonExample.cs

```diff
@@ -115,7 +114,7 @@ internal static IGeometry ClipPolygon(IGeometry polygon, IPolygonal clipPolygona
 
 
         [STAThread]
-        public static void Main(string[] args)
+        public static void main(string[] args)
```

Comment:
Most likely, it was because I hadn't done [this](https://github.com/NetTopologySuite/NetTopologySuite/pull/183/files#diff-d6393584118e2f28680d7a7d1c9e98f6R8) yet (copied over the `StartupObject` from the project's properties).

Reverting.